compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeBuilder.java

```diff
@@ -1568,6 +1560,20 @@ public BLangNode transform(FunctionBodyBlockNode functionBodyBlockNode) {
         return bLFuncBody;
     }
 
+    private List<BLangStatement> getStatementList(List<BLangStatement> statements,
```

Comment:
Still doesn't reflect the actual check right? I actually meant moving the check in the if condition to a separate method.